# Hekateros hekateros_control/OpenGripper service

---
